recipes/jupyterlab-pullrequests/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "jupyterlab-pullrequests" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 876b952d9cd3ee12e9d912fbf8800d8945621cccac5b99e04a5585dc7a71cce9
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - nodejs
```

Comment:
I'm pretty sure this one can be in host too.

Reply:
Thanks @ocefpaf - I made the change